packages/build/package.json

```diff
@@ -20,7 +20,9 @@
     "Swyx <swyx@netlify.com> (https://www.swyx.io)"
   ],
   "scripts": {
-    "prepublishOnly": "cd ../../ && npm run prepublishOnly"
+    "prepublishOnly": "cd ../../ && npm run prepublishOnly",
+    "prebuild": "rm -rf lib && mkdir lib",
```

Comment:
I think on a copy you don't need to create the dir `lib` you can remove that part on all the packages